### PR TITLE
DT-1557: ⬆️ Application insights v3

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -46,7 +46,6 @@ dependencies {
   implementation("org.apache.commons:commons-text:1.9")
   implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310")
   implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
-  implementation("com.pauldijou:jwt-core_2.11:4.3.0")
```
Comment (petergphillips) [inline build.gradle.kts:49]: not used

File: src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/integration/AppointmentLinkerIntegrationTest.kt [hunk 2]
```diff
@@ -18,9 +16,6 @@ import javax.transaction.Transactional
 @Transactional
 class AppointmentLinkerIntegrationTest : IntegrationTest() {
 
-  @Value("\${appointmentLinkerToken}")
-  private val appointmentLinkerToken: String? = null
```
Comment (petergphillips) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/integration/AppointmentLinkerIntegrationTest.kt:22]: move away from hard coded token in application properties

File: src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/integration/IntegrationTest.kt [hunk 3]
```diff
@@ -32,8 +31,8 @@ abstract class IntegrationTest {
   @Autowired
   lateinit var gson: Gson
 
-  @Value("\${token}")
-  protected val token: String? = null
+  @Autowired
+  lateinit var jwtAuthHelper: JwtAuthHelper
```
Comment (petergphillips) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/integration/IntegrationTest.kt:35]: instead use auth helper to dynamically generate the tokens

**Review by JamieDraperUK — APPROVED** [review]
LGTM

**Pull request merged**